updatecli/updatecli.d/blueocean.yaml

```
@@ -46,9 +46,9 @@ targets:
     spec:
       file: content/doc/book/installing/_docker-for-tutorials.adoc
       matchpattern: >-
-        (.*)jenkins-plugin-cli(.*)blueocean:(.*) (.*)
+        (.*)jenkins-plugin-cli(.*)blueocean:(.*?) (.*)\"
       replacepattern: >-
-        ${1}jenkins-plugin-cli${2}blueocean:{{ source "blueoceanLatestVersion" }} ${4}
+        ${1}jenkins-plugin-cli${2}blueocean:{{ source "blueoceanLatestVersion" }} ${4}"
```

I'm a little surprised at the trailing double quote without an escape.  Is that an intentional difference from the earlier change where an escaped double quote was added?
```suggestion
        ${1}jenkins-plugin-cli${2}blueocean:{{ source "blueoceanLatestVersion" }} ${4}"
```

---

My mistake.  The trailing double quote was already there before.  Bad code reading by me.